html/webappapis/scripting/reporterror.any.js

```diff
@@ -0,0 +1,22 @@
+setup({ allow_uncaught_exception:true });
+
```

Comment:
We should test `undefined` per https://github.com/whatwg/html/pull/1196#issuecomment-885054066